jax/_src/api.py

```diff
@@ -2328,7 +2328,47 @@ def _device_get(x):
   else:
     return copy()
 
-def device_get(x):
+def device_get(x: Any):
+  """Transfer ``x`` to host.
+
+  Args:
+    x: An array, scalar, DeviceArray or (nested) standard Python container thereof
+      representing the array to be transferred to host.
+
+  Returns:
+    An array or (nested) Python container thereof representing the
+    value of ``x``.
+  Examples:
+    Passing a DeviceArray:
+
+    >>> import jax
+    >>> devices = jax.local_devices()
+    >>> x = jax.numpy.array([1., 2., 3.])
+    >>> y = jax.device_put(x, device=devices[0])
```

Comment:
There is no need to call `device_put` on a jax array: it already lives on a device. You could skip `device_put`, or you could define `x` as a standard numpy array, in which case `device_put` puts it on the device.